Themis/Views/Exercises/ExerciseView.swift

```diff
@@ -64,6 +68,18 @@ struct ExerciseView: View {
                     searchButton
                 }
             }
+            ToolbarItem(placement: .navigationBarTrailing) {
```

Comment:
Maybe put the searchbar in the middle, since the toolbar now has 3 items when counting in the back button and this might look cleaner. So keep this trailing and make searchbar placement .principal.

Reply:
Ok as it is just one word to change I would leave it for now as is and we can discuss on Wednesday who prefers which.